app/routes/podcast/index.tsx

```diff
@@ -0,0 +1,215 @@
+import * as React from 'react'
+import {json, useRouteData} from 'remix'
+
+import clsx from 'clsx'
+import {Tab, TabList, TabPanel, TabPanels, Tabs} from '@reach/tabs'
+import {Link} from 'react-router-dom'
+import {Grid} from '../../components/grid'
+import {images} from '../../images'
+import {H2, H6} from '../../components/typography'
+import type {KCDLoader, MdxListItem} from '../../../types'
+import {AppleIcon} from '../../components/icons/apple-icon'
+import {externalLinks} from '../../external-links'
+import {RssIcon} from '../../components/icons/rss-icon'
+import {SpotifyIcon} from '../../components/icons/spotify-icon'
+import {GoogleIcon} from '../../components/icons/google-icon'
+import {ChevronDownIcon} from '../../components/icons/chevron-down-icon'
+import {BlogSection} from '../../components/sections/blog-section'
+import {articles} from '../../../storybook/stories/fixtures'
+import {
+  getMdxPagesInDirectory,
+  mapFromMdxPageToMdxListItem,
+} from '../../utils/mdx'
+import {FeaturedSection} from '../../components/sections/featured-section'
+
+type LoaderData = {
+  podcasts: Array<MdxListItem>
+  seasons: Array<number>
+}
+
+export const loader: KCDLoader = async () => {
+  // TODO: this should support the season dirs
+  const pages = await getMdxPagesInDirectory('podcast-next/01')
+
+  const data: LoaderData = {
+    podcasts: pages.map(mapFromMdxPageToMdxListItem),
+    seasons: [3, 2, 1],
+  }
+
+  return json(data)
+}
+
+export function meta() {
+  return {
+    title: 'Podcasts by Kent C. Dodds',
+    description: 'Get really good at making software with Kent C. Dodds',
+  }
+}
+
+function PodcastAppLink({
+  icon,
+  children,
+  ...props
+}: JSX.IntrinsicElements['a'] & {icon: React.ReactElement}) {
+  return (
+    <a
+      {...props}
+      className="focus-ring text-primary bg-secondary flex flex-none items-center mb-4 mr-4 px-8 py-4 rounded-full space-x-4"
+    >
+      <span className="text-gray-400">{icon}</span>
+      <span>{children}</span>
+    </a>
+  )
+}
+
+function PodcastHome() {
+  const data = useRouteData<LoaderData>()
+  console.dir({data})
+  // TODO: save state in url, like the filters on the blog?
+
+  const featured = data.podcasts[0]
+
+  return (
+    <div>
+      <Grid className="grid-rows-max-content mb-36 mt-16">
+        <div className="col-span-full mb-12 lg:col-span-6 lg:col-start-7 lg:row-span-2 lg:mb-0">
+          <img className="object-contain" {...images.kayak()} />
```

Comment:
This is spreading a string (the function returns what you should pass into the `src` prop.

```suggestion
          <img className="object-contain" src={images.kayak()} alt={images.kayak} />
```

Reply:
Got it. And `images.kayak.alt` :+1: